app/pages/project/classify.cjsx

```diff
@@ -132,6 +132,7 @@ module.exports = createReactClass
           currentClassifications.forWorkflow[props.workflow.id] = classification
           @setState { classification }
         .catch (error) =>
+          # console.error(error)
```

Comment:
might as well remove but nothing that blocks merging

Reply:
I left it commented in because I uncovered a bug that was being swallowed by just setting it in the react component state and I was going to come back to it later.